src/ScrapingInput.re

```diff
@@ -1,13 +1,5 @@
 open Chrome.Extensions;
 
-/** Convert value from event to string
```

Comment:
why was this moved from here to services? if anything, it needs to be under a new "utilities.re" file

Reply:
I can move it there, in the PR description I said it was extracted because its used across multiple components

Reply:
addressed in latest commit